### PR TITLE
[EDU-3145] feat: add block tor list

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new page on How to block Tor exit node IP addresses.

[EDU-3145]: https://aziontech.atlassian.net/browse/EDU-3145?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/secure-journey/advanced-configurations/block-tor-networks.mdx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+namespace: docs_secure_block_tor
+menu_namespace: secureMenu
+permalink: /documentation/products/guides/secure/block-tor-networks/
+---
+
+import Button from '~/components/Button.astro'
+
+Tor exit nodes are are the final point for the Tor network to connect with the internet. As Tor network traffic isn't encrypted anymore, data being accessed can become visible to the node and possibly result in security concerns or even malicious traffic.
+
+Azion provides the **Azion IP Tor Exit Nodes** network list to all users registered on Azion. This list can be used to configure a behavior using Rules Engine to block all requests coming from IPs contained in the network list.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/block-tor-networks.mdx:14]:
```suggestion
Azion provides the **Azion IP Tor Exit Nodes** network list to all users registered with Azion. This list can be used to configure a behavior using Rules Engine to block all requests coming from IPs contained in the network list.
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/block-tor-networks.mdx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+      "list_type": "ip_cidr",
+      "name": "Azion IP Tor Exit Nodes",
+      "country_list": [],
+      "ip_list": [
+        "192.168.0.5"
+      ]
+    }
+  ]
+}
+}
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/block-tor-networks.mdx:73]:
```suggestion
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/block-tor-networks.mdx [hunk 3]
```diff
@@ -0,0 +1,149 @@
+---
+title: How to block Tor exit node IP addresses
+description: a
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/block-tor-networks.mdx:3]:
```suggestion
description: The Azion IP Tor Exit Nodes network list can be used to configure a behavior using Rules Engine to block all requests coming from IPs contained in the network list.
```

**Review by bru-andrade — APPROVED** [review]
👍🏻

File: src/content/docs/en/pages/secure-journey/advanced-configurations/block-tor-networks.mdx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+description: The Azion IP Tor Exit Nodes network list can be used to configure a behavior using Rules Engine to block all requests coming from IPs contained in the network list.
+meta_tags: edge, secure, tor, exit nodes, list, rules engine, behavior
+namespace: docs_secure_block_tor
+menu_namespace: secureMenu
+permalink: /documentation/products/guides/secure/block-tor-networks/
+---
+
+import Button from '~/components/Button.astro'
+
+Tor exit nodes are are the final point for the Tor network to connect with the internet. As Tor network traffic isn't encrypted anymore, data being accessed can become visible to the node and possibly result in security concerns or even malicious traffic.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/block-tor-networks.mdx:12]:
```suggestion
**Tor exit nodes** are the final point for the **Tor network** to connect with the internet. As Tor network traffic isn't encrypted anymore, data being accessed can become visible to the node and possibly result in security concerns or even malicious traffic.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/secure-journey/advanced-configurations/block-tor-networks.mdx:12]: italics or bold? para deixar com énfase

File: src/content/docs/en/pages/secure-journey/advanced-configurations/block-tor-networks.mdx [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+<Button href="/en/documentation/products/edge-firewall/network-layer-protection/network-lists/" text="go to network lists reference" variant="secondary"></Button>
+
+---
+
+## Via RTM
+
+To create a rule:
+
+1. [On RTM](/en/documentation/products/guides/how-to-access-rtm/), navigate to **Edge Firewall**.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/block-tor-networks.mdx:24]:
```suggestion
1. [Access RTM](/en/documentation/products/guides/how-to-access-rtm/) > **Edge Firewall**.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**